--- libwebcam-0.2.5/libwebcam/pkgconfig/libwebcam.pc_base.orig	2014-11-29 11:11:28.000000000 +0100
+++ libwebcam-0.2.5/libwebcam/pkgconfig/libwebcam.pc_base	2025-09-07 15:22:00.513007824 +0200
@@ -1,6 +1,6 @@
 prefix=<--PREFIX--!>
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=${exec_prefix}/<--LIBDIR--!>
 includedir=${exec_prefix}/include
  
 Name: libwebcam
--- libwebcam-0.2.5/libwebcam/CMakeLists.txt.orig	2014-11-29 11:11:28.000000000 +0100
+++ libwebcam-0.2.5/libwebcam/CMakeLists.txt	2025-09-07 15:22:10.349621201 +0200
@@ -57,7 +57,7 @@ set_target_properties (webcam_static PRO
 
 # create libwebcam.pc (for pkg-config)
 execute_process (
-	COMMAND			sh -c "sed -e 's_<--PREFIX--!>_${CMAKE_INSTALL_PREFIX}_g' ${CMAKE_CURRENT_SOURCE_DIR}/pkgconfig/libwebcam.pc_base  > ${CMAKE_CURRENT_BINARY_DIR}/libwebcam.pc"
+	COMMAND			sh -c "sed -e 's_<--PREFIX--!>_${CMAKE_INSTALL_PREFIX}_g;s_<--LIBDIR--!>_${CMAKE_INSTALL_LIBDIR}_g' ${CMAKE_CURRENT_SOURCE_DIR}/pkgconfig/libwebcam.pc_base  > ${CMAKE_CURRENT_BINARY_DIR}/libwebcam.pc"
 	RESULT_VARIABLE		PC_EXECUTE_RESULT
 	ERROR_QUIET
 )
