Index: libwebcam/CMakeLists.txt
===================================================================
--- libwebcam/CMakeLists.txt	(revision 97)
+++ libwebcam/CMakeLists.txt	(working copy)
@@ -142,11 +160,6 @@
 	OUTPUT_STRIP_TRAILING_WHITESPACE
 	ERROR_QUIET
 )
-if (EXECUTE_RESULT)
-	message(FATAL_ERROR "Unable to determine current dpkg architecture: ${EXECUTE_RESULT}")
-else (EXECUTE_RESULT)
-	message("** Debian package architecture: ${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}")
-endif (EXECUTE_RESULT)
 
 # Generic
 set (CPACK_GENERATOR					"TBZ2;DEB")
Index: uvcdynctrl/CMakeLists.txt
===================================================================
--- uvcdynctrl/CMakeLists.txt	(revision 97)
+++ uvcdynctrl/CMakeLists.txt	(working copy)
@@ -111,11 +111,6 @@
 	OUTPUT_STRIP_TRAILING_WHITESPACE
 	ERROR_QUIET
 )
-if (EXECUTE_RESULT)
-	message(FATAL_ERROR "Unable to determine current dpkg architecture: ${EXECUTE_RESULT}")
-else (EXECUTE_RESULT)
-	message("** Debian package architecture: ${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}")
-endif (EXECUTE_RESULT)
 
 # Generic
 set (CPACK_GENERATOR					"TBZ2;DEB")
